### ISSUE TITLE
Failed to login with 2FA GA

### ISSUE THREAD

**Issue by Frodox** [a1phat0ny/instascrape#10]
**Description**
I'm failed to login to my account via

```
instascrape login

    Saved Cookies

(1) + [Login New Account]

(1-1)choice> 1
Username: $USER
Password: 
[2FA] Security Code: 711389
✖ HTTPError: 400 Client Error: Bad Request for url: https://i.instagram.com/api/v1/users/1382407697/info/
```
I'm using Google Authenticator for 2FA codes.

There is following content at https://i.instagram.com/api/v1/users/1382407697/info/
```
{
"message": "useragent mismatch",
"status": "fail"
}
```

**Information**
* OS: Mageia 6 (Linux)
* Version of `instascrape`: `v2.0.1`

**Comment by Agrafador** [comment]
Here is the solution: https://mpsocial.com/t/any-api-end-point-to-request-user-info-by-ig-id/86705

**Comment by a1phat0ny** [comment]
This bug's already been fixed in 3914987919da0c93af8ce1440b1b91dd2f1378a8. It will be included in the next release.
For now, you can install the latest commit of `instascrape` using pip.
`$ pip install git+https://github.com/a1phat0ny/instascrape.git@master`
